flow/alerting/classifier.go

```diff
@@ -261,6 +261,17 @@ func GetErrorClass(ctx context.Context, err error) (ErrorClass, ErrorInfo) {
 		}
 	}
 
+	// Reference:
+	// https://github.dev/jackc/pgx/blob/master/pgconn/pgconn.go#L733-L740
```

Comment:
```suggestion
	// https://github.com/jackc/pgx/blob/61d3c965ad442cc14d6b0e39e0ab3821f3684c03/pgconn/pgconn.go#L731
```
1. link doesn't need to grant oauth to github.dev
2. link uses hash so line reference doesn't drift